src/Validator.php

```diff
@@ -103,4 +103,33 @@ public function validate(string $attribute, $value, array $parameters, Validator
 
         return false;
     }
+    
+    /**
+     * Replace all place-holders for the required_with rule.
+     *
+     * @param  string  $message
+     * @param  string  $attribute
+     * @param  string  $rule
+     * @param  array   $parameters
+     * @return string
+     */
+    public function replacer($message, $attribute, $rule, $parameters, ValidatorContract $validator)
```

Comment:
The exception is being caused by the `replacer` method which obviously takes 5 arguments while Laravel <= 5.3 passes only 4 arguments to this method. (Reference: https://github.com/illuminate/validation/blob/5.3/Validator.php#L2217)

I suggest getting rid of the 5th parameter to keep the backwards compatibility.

Reply:
We can get rid of the 5th parameter if we are able to get the request parameters without it, but I have not been able to get the parameters without setting the `$this->setRequest($validator)` part in `replacer()`.

Do you have a proposed way of getting the country value?

Reply:
When i wrote this line comment i meant to pinpoint the source of the issue, and continue with the follow-up comment in the line 116